frontend/src/Student/components/ShowCard.js

```diff
@@ -18,86 +19,160 @@ const Card = styled.div`
+      const icon =
+        entry.provider === 'youtube' ? (
+          <FaYouTube size='3em' />
+        ) : (
+          <FaVimeo size='3em' />
+        )
+      return (
+        <a href={url} target='_blank'>
+          <EntryNoThumbContainer>
+            {icon}
+            <h5>{entry.title}</h5>
+          </EntryNoThumbContainer>
+        </a>
+      )
+    case 'OTHER':
+      return (
         <EntryNoThumbContainer>
-          {icon}
+          <FaBook size='3em' />
           <h5>{entry.title}</h5>
         </EntryNoThumbContainer>
-      </a>
-    case 'OTHER':
-      return <EntryNoThumbContainer>
-        <FaBook size='3em' />
-        <h5>{entry.title}</h5>
-      </EntryNoThumbContainer>
+      )
     default:
       return null
   }
 }
 
-const ShowCard = (props) => (
+const NewSubmission = ({ show }) => (
+  <Col
+    style={{ minHeight: '10em' }}
+    md={show.entries.length > 0 ? '3' : null}
+    className='text-center align-self-center d-flex justify-content-center align-items-center'
+  >
+    <Link to={`/submit?to=${show.id}`}>
+      <FaPlusCircle size='3em' />
+      <h5 className='mt-1'>New Submission</h5>
+    </Link>
+  </Col>
+)
+
+const SubmittedEntries = ({ show }) =>
+  show.entries.map(entry => (
+    <Col
+      md='3'
+      className='text-center align-self-center d-flex justify-content-center align-items-center'
+      style={{ minHeight: '10em' }}
+      title={entry.title}
+      key={entry.id}
+    >
+      <EntryContainer>
+        <EntryThumb entry={entry} />
+        {// If after entry end and before judging end, display "judging phase" , else display accepted or denied
+          moment().isBetween(show.entryEnd, show.judgingEnd) ? (
+            <JudgingPhase>In Judging Phase</JudgingPhase>
+          ) : moment().isAfter(show.judgingEnd) ? (
+            entry.invited ? (
+              <Accepted>Invited</Accepted>
+            ) : (
+              <NotAccepted>Not Invited</NotAccepted>
+            )
+          ) : null }
+      </EntryContainer>
+    </Col>
+  ))
+
+const AcceptingSubmissionMessage = ({ show }) => (
+  <div>
+    Accepting Submissions Until:{' '}
+    <Moment format='MMMM Do YYYY'>{show.entryEnd}</Moment>
+  </div>
+)
+
+const ShowCard = props => (
   <Card>
     <Row>
       <Col style={{ display: 'flex' }}>
         <h2 style={{ alignSelf: 'flex-end', margin: 0 }}>{props.show.name}</h2>
       </Col>
       <Col className='text-right'>
         <div>
-          <h5>{props.show.entries.length}/{props.show.entryCap} Submissions</h5>
+          <h5>
+            {props.show.entries.length}/{props.show.entryCap} Submissions
+          </h5>
         </div>
         <div>
-          Accepting Submissions Until: <Moment format='MMMM Do YYYY'>{props.show.entryEnd}</Moment>
+          {moment().isAfter(moment(props.show.entryEnd)) ? (
+            'Submissions are closed'
+          ) : (
+            <AcceptingSubmissionMessage {...props} />
```

Comment:
I don't think you need to break this out into a separate component. Consider moving the ternary up one level in the DOM tree, and return the parent `<div>` from both "components".